ecospheres_migrator/migrator.py

```diff
@@ -68,41 +71,47 @@ def transform(self, transformation: Path, selection: list[Record]) -> TransformB
         for r in selection:
             log.debug(f"Processing {r.uuid} (template={r.template} state={r.state})")
             original = self.gn.get_record(r.uuid)
+            batch_record = TransformBatchRecord(
+                url=self.gn.url,
+                uuid=r.uuid,
+                template=r.template,
+                state=r.state,
+                original=xml_to_string(original),
+            )
             if r.state and r.state.stage == WorkflowStage.WORKING_COPY:
                 batch.add(
                     FailureTransformBatchRecord(
```

Comment:
Would this make more sense as a "skipped" now that we have it?

Reply:
Yep, I'll do a separate PR to fix #53